deepface/modules/preprocessing.py

```diff
@@ -2,12 +2,14 @@
 from typing import Union, Tuple
 import base64
 from pathlib import Path
-
 # 3rd party
 import numpy as np
 import cv2
 from PIL import Image
 import requests
+from deepface.modules.kyc_validation import extract_frame_with_face
```

Comment:
what does kyc mean?

Reply:
Know Your Client. KYC or KYC check is the mandatory process of identifying and verifying the client's identity when opening an account and periodically over time. In other words, banks must ensure that their clients are genuinely who they claim to be.